ui/app/templates/vault/cluster/unseal.hbs

```diff
@@ -31,21 +31,29 @@
           is
           {{if this.model.unsealed "unsealed" "sealed"}}
         </p>
-        <p>
-          Unseal Vault by entering portions of the unseal key. This can be done via multiple mechanisms on multiple
-          computers. Once all portions are entered, the root key will be decrypted and Vault will unseal.
-        </p>
+        {{#if this.model.unsealed}}
+          <p>Please wait while we redirect you.</p>
+        {{else}}
+          <p>
+            Unseal Vault by entering portions of the unseal key. This can be done via multiple mechanisms on multiple
+            computers. Once all portions are entered, the root key will be decrypted and Vault will unseal.
+          </p>
+        {{/if}}
+        {{#unless this.model.unsealed}}
+          <Shamir::Flow
+            @action="unseal"
+            @onLicenseError={{action "handleLicenseError"}}
+            @onShamirSuccess={{action "transitionToCluster"}}
+            @updateProgress={{action "reloadCluster"}}
+            @checkComplete={{action "isUnsealed"}}
+            @buttonText="Unseal"
+            @isComplete={{action "isUnsealed"}}
+            @threshold={{this.model.sealThreshold}}
+            @progress={{this.model.sealProgress}}
+            class="has-top-margin-m"
```

Comment:
can we add this class directly to the Shamir::Form (where it looks like this gets passed in)? Or does this class only pertain to the flow component?

Reply:
During the styling discussion at the offsite, we decided margin and padding classes like this should be passed in to keep the component very generic and easy to fit wherever you need it to go